app/assets/javascripts/controllers/angular-bootstrap/DatepickerCtrl.js

```diff
@@ -0,0 +1,22 @@
+miqAngularApplication.controller('DatepickerCtrl', function ($scope) {
+  $scope.minDate = new Date();
+
+  $scope.disabled = function(date, viewDate, mode) {
+    disabledDate = new Date($scope.$parent.angularForm[viewDate].$viewValue);
+
+    if($scope.$parent.angularForm[viewDate].$untouched) {
+      return ( mode === 'day' && (disabledDate.getUTCDate() === date.getDate()) &&
+                              disabledDate.getUTCMonth() === date.getMonth() &&
+                              disabledDate.getUTCFullYear() === date.getFullYear())
```

Comment:
Since the code above has been removed as it was never used, I guess more discussion on this would be pointless.
```moment.js```has been added as a gem, hopefully that should make it easier for us to deal with ```Date``` and ```TZ``` operations.